apps/mark-scan/backend/src/audio/outputs.ts

```diff
@@ -0,0 +1,59 @@
+/* istanbul ignore file */
+import { LogEventId, Logger } from '@votingworks/logging';
+import { exec } from '@votingworks/backend';
+import { NODE_ENV } from '../globals';
+
+const VX_UI_USERNAME = 'vx-ui';
+const CMD_RUN_AS_VX_UI = `sudo -u ${VX_UI_USERNAME} XDG_RUNTIME_DIR=/run/user/$(id -u ${VX_UI_USERNAME})`;
```

Comment:
we probably need to add something to the sudoers file in complete-system for this to work on a prod image cc @amcmanus

Reply:
Yeah, worth double-checking - Adam, for context, these commands will be running in the mark-scan backend service (vx-services). Using a similar pattern to the `pactl.sh` script we execute for the headphones check , but running the command directly instead of prxying through a script - LMK if there could be any issues there

Reply:
To make `CMD_RUN_AS_VX_UI` work, we'll need to make a couple of changes to our `sudoers` config. 

To enable setting the `XDG_RUNTIME_DIR` env var, we need to add 
```
Defaults!/usr/bin/amixer env_keep += "XDG_RUNTIME_DIR"
```

To allow the vx-services user to run amixer as the vx-ui user, we need to add
```
vx-services ALL=(vx-ui:ALL) NOPASSWD: /usr/bin/amixer
```

With that said, it may not be necessary to pass the `XDG_RUNTIME_DIR` env var to amixer commands. That was specific to the `pactl` command to gain access to the pulseaudio config running under the `vx-ui` user, and a reason for using a script rather than a similar sudo config as above. 

I think we should get your code running on a VSAP (not via VxDev) and test both the `sudoers` and `amixer` configs. It's not really possible for me to simulate the audio config on any hardware other than a VSAP, and I want to be sure we're not missing something specific to a production environment. I think there's a chance `amixer` may need to include the `Master` audio channel based on behavior we saw with the startup chime and headphone support. (My primary concern is that the alarm could then put the system in a state where headphone audio is no longer audible.)

Reply:
Ah, thanks - I see the corresponding suoders config for `pactl.sh` now. Was only able to test the "run as vx-ui" piece while logged in as vx-admin on the prod vsap, so will try just copying this code over and running the actual app to make sure it all works end-to-end. Good to know we can probably do this without setting `XDG_RUNTIME_DIR` - will drop it for the test to verify